File: Tareas/Unidad2/a2253332351_practica04/34_InteresSimple.java
import java.util.Scanner;

public class InteresSimple {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double capital;
        double tasa;
        double tiempo;
        double interes;
        double monto;

        System.out.print("Ingrese el capital inicial: ");
        capital = entrada.nextDouble();
        System.out.print("Ingrese la tasa de interés anual (%): ");
        tasa = entrada.nextDouble();
        System.out.print("Ingrese el tiempo en años: ");
        tiempo = entrada.nextDouble();

        interes = (capital * tasa * tiempo) / 100;
        monto = capital + interes;

        System.out.println("El interés generado es: " + interes);
        System.out.println("El monto final es: " + monto);
    }
}

File: Tareas/Unidad2/a2253332351_practica04/34_InteresSimple.py
# InteresSimple.py

capital = float(input("Ingrese el capital inicial: "))
tasa = float(input("Ingrese la tasa de interés anual (%): "))
tiempo = float(input("Ingrese el tiempo en años: "))

interes = (capital * tasa * tiempo) / 100
monto = capital + interes

print(f"El interés generado es: {interes}")
print(f"El monto final es: {monto}")
